app/controllers/sessions_controller.rb

```diff
@@ -35,4 +19,56 @@ def destroy
     # flash[:notice] = "Вие излязохте успешно от системата."
     redirect_to root_path
   end
+
+  def facebook
+    omniauth_response = request.env['omniauth.auth']
+    facebook_email = omniauth_response['info']['email'] if omniauth_response.is_a?(Hash)
+
+    user = User.where(email: facebook_email).last
```

Comment:
Can we extend the login to also create a new user account if one doesn't exist yet?